lib/src/fixed.dart

```diff
@@ -35,30 +35,26 @@ class Fixed implements Comparable<Fixed> {
   /// If the [amount] isn't valid then
   /// a [FixedParseException] is thrown.
   ///
-  /// If [invertSeparator] = false then we
-  /// assume '.' is the decimal place and ',' is the group separator.
-  ///
-  /// If [invertSeparator] = true then we
-  /// assume ',' is the decimal place and '.' is the group separator.
+  /// [decimalSeparator] and [groupSeparator] are optional and
+  /// default to '.' and ',' respectively.
   factory Fixed.parse(
     String amount, {
     int? scale,
```

Comment:
given the breaking changes I would like to change scale to decimalDigits to bring it inline with Money2.